### ISSUE TITLE
Virtual function documentation using Sphinx

### ISSUE THREAD

**Issue by Revbr** [michaeljones/breathe#600]
1. I tried to document a C++ code using Sphinx, breathe and doxygen. The pdf doc generated in doxygen is coming correctly for virtual functions, but the pdf document generated using Sphinx(using doxygen xml) is just showing 
void functioname()
instead of 
virtual void functionname().

2. what is the doxygen directive for documenting overlaoded functions? I am getting error even after adding the arguments.

**Comment by vermeeren** [comment]
@Revbr 

1. Can you check if this problem also occurs with HTML output? Is it pure virtual or just virtual? If I check https://breathe.readthedocs.io/en/latest/doxygen.html the `testMeToo` doesn't show up properly with the `virtual` prefix too. Could be this needs to be added in Breathe or in Sphinx C/C++ domain. I do see some logic for adding `virtual` in `breathe/renderer/sphinxrenderer.py`, not entirely sure at this point.

```cpp
    //! A pure virtual member.
    /*!
      \sa testMe()
      \param c1 the first argument.
      \param c2 the second argument.
    */
    virtual void testMeToo(char c1,char c2) = 0;
```

![image](https://user-images.githubusercontent.com/5957454/99195620-0ef5f380-2787-11eb-9c1c-b9246eb2c2a7.png)

2. Should be possible, but the syntax can be a bit finicky. https://github.com/michaeljones/breathe/issues/5#issuecomment-72326037 I remember seeing some other posts with examples in the issue tracker, but I cannot find them easily currently. Can you share function declaration?

**Comment by vermeeren** [comment]
The first point was just resolved thanks to #628. Since I never received a reply to the second point I will assume that is resolved too as it seems somewhat support-y.

Closing this for now.
